src/XMakeBuildEngine/UnitTestsPublicOM/Definition/Project_Tests.cs

```diff
@@ -2415,6 +2416,251 @@ public void ImportPropertyEvaluatingToInvalidPath()
             }
            );
         }
+
+        [Fact]
+        public void GetItemProvenanceOnlyStringLiteral()
+        {
+            var project =
+                @"<Project ToolsVersion='msbuilddefaulttoolsversion' DefaultTargets='Build' xmlns='msbuildnamespace'>
+                  <ItemGroup>
+                    <A Include=`1;2;3`/>
+                    <B Include=`1;2;3` Exclude=`1`/>
+                    <C Include=`2;3` Exclude=`2`/>
```

Comment:
What's the expected behavior if this had been Exclude=1? In other words, does the API return "any item element in which this path was mentioned" or "any item element in which this path could have been excluded"?